activate.ps1

```diff
@@ -11,7 +11,7 @@ if ($MyInvocation.CommandOrigin -eq 'runspace') {
     exit 1
 }
 
-function tye() {
+function dtye() {
```

Comment:
dtye or dev-tye?

Reply:
`dtye` is better because then I don't have to rename my script.